application/src/main/java/run/halo/app/security/session/ReactiveFindByIndexNameSessionRepository.java

```diff
@@ -0,0 +1,24 @@
+package run.halo.app.security.session;
+
+import java.util.Map;
+import org.springframework.session.FindByIndexNameSessionRepository;
+import org.springframework.session.PrincipalNameIndexResolver;
+import org.springframework.session.Session;
+import reactor.core.publisher.Mono;
+
+/**
+ * TODO: Wait for <code>spring session 3.3 released</code> and then remove this interface.
+ * if you want to remove this interface, please modify the {@link PrincipalNameIndexResolver} in
+ * {@link InMemoryReactiveIndexedSessionRepository} to have a parameterized constructor(since
+ * spring session 3.3) and overload the findByPrincipalName method.
+ */
+public interface ReactiveFindByIndexNameSessionRepository<S extends Session> {
```

Comment:
Hi @guqing ，I have proposed another PR to [upgrade Spring Boot to 3.3.0-RC1](https://github.com/halo-dev/halo/pull/5760) which includes Spring Session 3.3.

The interface can be removed after merging https://github.com/halo-dev/halo/pull/5760.

Reply:
ok